### ISSUE TITLE
Why there is a gap between the zone?

### ISSUE THREAD

**Issue by zismylove** [bw2012/UE4VoxelTerrain#11]
Why there is a gap between the zone?

**Comment by bw2012** [comment]
Hi
Could you please download screenshot?

**Comment by zismylove** [comment]
http://imgur.com/cJFIbRS

**Comment by bw2012** [comment]
It is really strange. I have no gaps on my PC.
So I need more information:
- which platform? (is it PC or Mac?)
- which version of plugin? (do you use pre-released version 0.0.8? or just clone of current master branch?)
- is it LOD enabled?

**Comment by zismylove** [comment]
yes,is pc .and the version is 0.0.8,download for your githup main page.this is download link http://bit.ly/2mg19mE.   and look at this,http://imgur.com/pODrwSq    
when is set terrain size more then 5,it will happen.

**Comment by bw2012** [comment]
ok, thanks for information 
I will fix it asap

**Comment by bw2012** [comment]
and one more question: is it problem exist if terrain size is 5?

**Comment by zismylove** [comment]
no,if terrain size is 5,is ok.

**Comment by zismylove** [comment]
http://imgur.com/RqE7A5G

**Comment by zismylove** [comment]
the cat is cute,i like it.

**Comment by bw2012** [comment]
Thanks!
I have released version 0.0.9 and check it with terrain size is 7. There is no gap on my PC.
Could you please check with this version?

**Comment by zismylove** [comment]
http://imgur.com/Cjf6Ytt, no change,when the terrain size more then 5.Perhaps the different  engine version caused this?

**Comment by bw2012** [comment]
Unfortunately it is still not reproducible on my PC with any terrain size.
But I don't give up and keep trying to find cause this strange gaps. It looks like some kind of float calculation error. At this moment I try to refactor LOD system. I hope this helps to solve the problem

**Comment by zismylove** [comment]
Thank you for your hard work.waiting for your good news.
